internal/cmd/users.go

```diff
@@ -452,3 +507,19 @@ func hasAccessToChangePasswordsForOtherUsers(client *api.Client, config *ClientH
 
 	return len(grants.Items) > 0, nil
 }
+
+func hasMinimumCount(min int, password string, check func(rune) bool) bool {
+	var count int
+	for _, r := range password {
+		if check(r) {
+			count++
+		}
+	}
+	return count >= min
+}
+
+// list of valid special chars is from OWASP, wikipedia
+func isValidSymbol(letter rune) bool {
+	match, _ := regexp.MatchString(fmt.Sprintf(`(.*[ !"#$%%&'()*+,-./\:;<=>?@^_{}|~%s%s]){1,}`, regexp.QuoteMeta(`/\[]`), "`"), string(letter))
```

Comment:
weird that we limit valid symbols, no?

Reply:
Yea, this list was based off of a bunch of sources that listed "allowed symbols for passwords"

Reply:
this list is too American.